server/src/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -613,6 +614,10 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
             sc.setParameters("username", username);
         }
 
+        if (keyword!= null) {
+            sc.setParameters("username",  "%" + keyword + "%");
+        }
+
         Pair<List<VpnUserVO>, Integer> result = _vpnUsersDao.searchAndCount(sc, searchFilter);
```

Comment:
What if both username and keyword are set? Shouldn't we prefer the username? Also what is `keyword` anyways?